html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <div>
+        <div class="Email">
+          <label for="Email">Email</label>
+          <br/>
+          <input type="text" name="email" id="email" required>
+        </div>
+        <div>
+          <div>
+            <label for="Question">Why is coding so cool?</label>
+            <br/>
+            <select name="dropdown">
+              <option value="value1">Select an option</option>
+              <option value="value2">Because it's interesting</option>
+              <option value="value3">Because it's fun</option>
+              <option value="value4">Because it's powerful</option>
+            </select>
+            <br/>
+            <br/>
+          </div>
+
+          <div>
+            <label for="FavoriteLanguage"> Favorite Language (choose one)</label>
+            <br/>
+            <br/>
+            <input type="radio" name="FavoriteLanguage" value="HTML"/>HTML
+            <input type="radio" name="FavoriteLanguage" value="CSS"/>CSS
+            <input type="radio" name="FavoriteLanguage" value="JAVASCRIPT"/>JAVASCRIPT
+          </div>
+          <div>
+            <p>Would you like to sign up for more information?</p>
+            <input type="checkbox" name="sign up" value>
+            <label for="Subscription"> Sign me up!</label>
```

Comment:
This is not properly configured. The `for` attribute` needs to match the `id` attribute on the form control element this is labeling.